### PR TITLE
Upgrade dependencies with important security updates

### PR DESCRIPTION
**Pull request by Haard30**
[CG] Updating 2 dependencies with important security updates:

1. `"Microsoft.IdentityModel.JsonWebTokens" Version="7.5.0"`: Updating to latest stable version.
2. `"System.Data.SqlClient" Version="4.8.6"`: Explicitly specifying version so that it takes precedence over any transitive dependency version declarations. ( In this case, `Microsoft.VisualStudio.Services.Client` whose latest version (like current) uses `4.8.5` but allows > `4.8.5`)

Testing
Tested following commands on both mac and windows:

azureauth --help
azureauth aad --help
azureauth ado pat --help
azureauth ado token --help
azureauth ado pat scopes --help
azureauth info --help
azureauth ado token
azureauth ado pat
azureauth aad (with different output and auth modes)

### CODE REVIEW

**Review by shalinikhare27 — APPROVED** [review]

File: src/MSALWrapper/MSALWrapper.csproj [hunk 1]
```diff
@@ -31,7 +31,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.4" />
     <PackageReference Include="Microsoft.Identity.Client.Extensions.Msal" Version="2.32.0" />
-    <PackageReference Include="Microsoft.IdentityModel.JsonWebTokens" Version="6.16.0" />
+    <PackageReference Include="Microsoft.IdentityModel.JsonWebTokens" Version="7.5.0" />
```
Comment (mijpeterson) [inline src/MSALWrapper/MSALWrapper.csproj:34]: Given that this is a major version bump, is there any testing we need to do to ensure this still works as expected? Major version bumps often come with breaking changes.

Comment (Haard30) [reply src/MSALWrapper/MSALWrapper.csproj:34]: I did our regular testing (mentioned in description). Also had a look at changelog of 7x release, did not find anything that can break anything. Here's the changelog https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/wiki/IdentityModel-7x

Comment (Haard30) [reply src/MSALWrapper/MSALWrapper.csproj:34]: We have an option to do a minor version bump here `6.35.0` which was release in Jan this year https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/releases/tag/6.35.0. But 7x has important bug fixes and performance improvement.

Comment (mijpeterson) [reply src/MSALWrapper/MSALWrapper.csproj:34]: If we've tested and confirmed it looks good, then I'm more than happy bumping to latest :). Approving!

**Review by mijpeterson — APPROVED** [review]

**Pull request merged**